docs/README.md

```diff
@@ -179,7 +179,7 @@ Metabase's reference documentation.
 ### Tools
 
 - [Tools overview](./usage-and-performance-tools/start.md)
-- [Auditing tools](./usage-and-performance-tools/audit.md)
+- [Metabase analytics](./usage-and-performance-tools/metabase-analytics.md)
```

Comment:
There is something interesting about the name of this feature. While in-product the collection is `Metabase analytics`, externally we will call it `Usage analytics` AFAIK.

I think this makes sense because this is called Usage Analytics as a feature, but as a Metabase collection, it would be ambiguous to call it `Usage analytics`